Vagrantfile

```diff
@@ -0,0 +1,25 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$script = <<SCRIPT
+sudo apt-get update
+sudo apt-get dist-upgrade -y
+sudo apt-get install gcc gcc-arm-none-eabi cmake cppcheck vera++ python -y
```

Comment:
This is a duplication of `tools/apt-get-install-deps.sh`. Why not reuse that? As core developers are not using Vagrant, this list will go easily bitrotten.